### PR TITLE
Updated changes as per the Laura Pacilio comments

### PR DESCRIPTION
**Pull request by jayaprakashnarayana**

### CODE REVIEW

**Comment by crw** [comment]
Thanks for the submission, @jayaprakashnarayana. Is there another place where the conversation with Laura occurred? Could you please link it here so that we can inspect the context of this submission? Thanks!

**Comment by jayaprakashnarayana** [comment]
> Thanks for the submission, @jayaprakashnarayana. Is there another place where the conversation with Laura occurred? Could you please link it here so that we can inspect the context of this submission? Thanks!

https://github.com/hashicorp/terraform/pull/29529#issuecomment-1149164700 here  is previous commit I sent for review

File: website/docs/cli/commands/destroy.mdx [hunk 1]
```diff
@@ -29,7 +29,7 @@ terraform apply -destroy
 For that reason, this command accepts most of the options that
 [`terraform apply`](/cli/commands/apply) accepts, although it does
 not accept a plan file argument and forces the selection of the "destroy"
-planning mode.
+planning mode. This includes the -target option which lets you specify a subset of resources that Terraform should destroy.
```
Comment (mgarrell777) [inline website/docs/cli/commands/destroy.mdx:32]:
```suggestion
planning mode. These options include the `-target` option which lets you specify a subset of resources that Terraform should destroy.
```

Just a slight suggested wording change from "This includes" to "These options include" to add clarity.

**Review by mgarrell777 — APPROVED** [review]
Hi @jayaprakashnarayana I'm Matthew, new technical writer for Terraform working with Laura Pacilio. Thank you for your efforts on this PR. I have 1 small wording change suggestion. But otherwise, it looks good to me.

**Review by jayaprakashnarayana — COMMENTED** [review]
committed the proposed changes.

**Pull request closed without merging**